CommandLineParams.java

```diff
@@ -0,0 +1,14 @@
+/**
+* output parameters from command line in reverse order 

```

Comment:
Asterisks are still not aligned. Take a look for example and fix everywhere http://www.oracle.com/technetwork/articles/java/index-137868.html